frame/composable-traits/src/dex.rs

```diff
@@ -101,6 +105,88 @@ pub trait Amm {
 	) -> Result<Self::Balance, DispatchError>;
 }
 
+/// Pool Fees
+#[derive(
+	Encode, Decode, MaxEncodedLen, TypeInfo, Clone, Default, PartialEq, Eq, Copy, RuntimeDebug,
+)]
+pub struct Fee<AssetId, Balance> {
+	// total fee
+	pub fee: Balance,
+	/// Amount of the fee pool charges for the exchange, this goes to liquidity providers.
+	pub lp_fee: Balance,
+	/// Amount of the fee that goes out to the owner of the pool
+	pub owner_fee: Balance,
+	/// Amount of the protocol fees(for PBLO holders) out of owner_fees.
+	pub protocol_fee: Balance,
```

Comment:
If I understand correctly, owner fee will  be 20% and protocol fee will be 100% here?

Reply:
Yes!

Reply:
So just to confirm, in above setting 20% of lp_fee will be gone to owner_fee and 100% of that goes to PBLO holders right?

Reply:
You are correct. To clarify and just to define terms, we can call the total fee = trading fee.
Then out of trading fee 80% is `lp_fee`, 20% is separated as `owner_fee`. At the beginning 100% of the `owner_fee` is allocated as `protocol_fee` which goes to PBLO holders.